runtime/src/lib.rs

```diff
@@ -329,10 +327,22 @@ impl pallet_simple_nft::Config for Runtime {
     type RoleKey = Role;
     type TokenMetadataKey = [u8; 32];
     type TokenMetadataValue = MetadataValue<Self::TokenId>;
+    type ProcessValidator = ProcessValidation;
     type WeightInfo = pallet_simple_nft::weights::SubstrateWeight<Runtime>;
     type MaxMetadataCount = MaxMetadataCount;
 }
 
+impl pallet_process_validation::Config for Runtime {
+    type Event = Event;
+    type ProcessIdentifier = [u8; 32];
+    type CreateProcessOrigin = EnsureRoot<AccountId>;
+    type DisableProcessOrigin = EnsureRoot<AccountId>;
+    type WeightInfo = pallet_process_validation::weights::SubstrateWeight<Runtime>;
+    type RoleKey = Role;
+    type TokenMetadataKey = [u8; 32];
```

Comment:
could use a common type to match with `pallet_simple_nft`

`TokenMetadataValue` too